src/main/java/io/codelex/flightplanner/flightplan/FlightPlanRepository.java

```diff
@@ -0,0 +1,54 @@
+package io.codelex.flightplanner.flightplan;
+
+import io.codelex.flightplanner.flightplan.domain.Flight;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Repository
+public class FlightPlanRepository {
+
+    private List<Flight> requests;
+    private final AtomicInteger idsGiven;
```

Comment:
nosaukums idsGiven liek domāt, ka mainīgais satur kaut kādus vairākus ID, lai gan patiesībā tas vienkārši ir pēdējais/nākamais id, tāpēc labāk nosauc to par previousId vai nextId